src/main/java/agh/ddd/groups/group/Group.java

```diff
@@ -1,11 +1,70 @@
 package agh.ddd.groups.group;
 
+import agh.ddd.groups.group.events.*;
+import agh.ddd.groups.group.valueobjects.GroupConfiguration;
+import agh.ddd.groups.group.valueobjects.GroupId;
 import org.axonframework.eventsourcing.annotation.AbstractAnnotatedAggregateRoot;
+import org.axonframework.eventsourcing.annotation.AggregateIdentifier;
+import org.axonframework.eventsourcing.annotation.EventSourcedMember;
+import org.axonframework.eventsourcing.annotation.EventSourcingHandler;
+
+import java.util.HashSet;
+import java.util.Set;
 
 /**
  * Created by mikolaj on 08.01.15.
  */
 
 public class Group extends AbstractAnnotatedAggregateRoot {
+    @AggregateIdentifier
+    private GroupId id;
+    private GroupConfiguration configuration;
+    private Set<Member> members = new HashSet<Member>();
```

Comment:
Pewności nie masz - refleksją możesz wstrzyknąć coś nawet do pola oznaczonego final.
http://stackoverflow.com/questions/3301635/change-private-static-final-field-using-java-reflection
To tak apropo. Ale użycie tutaj final jest jak najbardziej poprawne.